Sources/SPMTestSupport/Toolchain.swift

```diff
@@ -94,28 +94,4 @@ extension UserToolchain {
         return true
       #endif
     }
-
-    /// Helper function to determine whether serialized diagnostics work properly in the current environment.
-    public func supportsSerializedDiagnostics(otherSwiftFlags: [String] = []) -> Bool {
```

Comment:
not needed?

Reply:
That's the hope/goal since this was needed because we were using the old driver in CI contexts (which apparently has a bug triggered by some of our tests).